src/app/applications/application-add-edit/application-add-edit.component.html

```diff
@@ -119,16 +119,6 @@ <h1><span class="text-muted">Crown land File: {{application['clFile']}} &nbsp;&r
       <fieldset name="decisionForm" id="appDecision">
         <legend class="mb-3">Application Decision</legend>
         <div class="mb-4" *ngIf="application.decision">
-          <div class="form-group">
-            <label>Description</label>
-            <textarea class="form-control" rows="3" name="decisionDesc" id="decisionDesc" required aria-required="true" #decisionDesc="ngModel"
-              [ngClass]="{'is-invalid': isSubmitSaveClicked && decisionDesc.invalid}"
-              [(ngModel)]="application.decision.description">
-            </textarea>
-            <div class="invalid-feedback bg-danger" *ngIf="isSubmitSaveClicked && decisionDesc.invalid">
-              Decision Description is required
-            </div>
-          </div>
           <div class="form-group mb-0">
```

Comment:
May need to remove mb-0 (margin bottom 0) here, same as form-group you deleted on line 122.

Reply:
Removed!  Rebased so its still just 1 commit.